lib/assigner/include/vm_host.hpp

```diff
@@ -63,63 +62,93 @@ class VMHost : public evmc::Host
     }
 
     evmc::bytes32 get_storage(const evmc::address& addr,
-                              const evmc::bytes32& key) const noexcept final
+                              const evmc::bytes32& key) noexcept final
     {
-        const auto account_iter = accounts.find(addr);
-        if (account_iter == accounts.end())
-            return {};
-
+        auto account_iter = accounts.find(addr);
+        if (account_iter == accounts.end()) {
+            account_iter = get_account(addr);
```

Comment:
Maybe move it inside `get_account`? I see that you are going to override it, but `accounts` field is protected anyway, so this should not be a problem.